driver/ppm_fillers.c

```diff
@@ -2722,6 +2727,32 @@ static int f_sys_poll_x(struct event_filler_arguments *args)
 	return add_sentinel(args);
 }
 
+#define PPM_MS_MGC_MSK 0xffff0000
+#define PPM_MS_MGC_VAL 0xC0ED0000
+
+static int f_sys_mount_e(struct event_filler_arguments *args)
+{
+	unsigned long val;
+	int res;
+	u32 j;
+
+	/*
+	 * Like { PPM_AUTOFILL, 4, APT_REG, {{0}, {1}, {2}, {3} } }
+	 * but fix flags in arg 3.
+	 * See http://lxr.free-electrons.com/source/fs/namespace.c?v=4.2#L2650
+	 */
+	for (j = 0; j < 4; j++) {
+		syscall_get_arguments(current, args->regs, j, 1, &val);
+		if (j == 3 && (val & PPM_MS_MGC_MSK) == PPM_MS_MGC_VAL)
+			val &= ~PPM_MS_MGC_MSK;
+		res = val_to_ring(args, val, 0, true, 0);
```

Comment:
Use flag `true` only for pointers to user memory, integers are passed by values. Also, in this case we put the argument on the exit event, because it's less likely to get a page fault.